pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -15,9 +15,39 @@ rec {
         --add-flags "-classpath $gradle_launcher_jar org.gradle.launcher.GradleMain"
     '';
 
-    phases = "unpackPhase installPhase";
+    fixupPhase =
+      let
+        arch =
```

Comment:
If you plan on expanding this, changing it to a map lookup may be preferrable, something like
```
{ "x86_64-linux" = "linux-amd64"; 
  "i686-linux" = "linux-i386";
}."${stdenv.system}" or ""
```